.gitignore

```diff
@@ -1,3 +1,5 @@
 *.swp
+# IntelliJ module files (.idea folder, *.iml module files):
 .idea
+*.iml
```

Comment:
These changes aren't related to this change, would you mind reverting them?

Reply:
Sure, I'll open a separate PR.